includes/class-ckwc-integration.php

```diff
@@ -835,15 +851,6 @@ public function enqueue_styles() {
 				wp_enqueue_style( 'ckwc-sync-past-orders', CKWC_PLUGIN_URL . 'resources/backend/css/sync-past-orders.css', array(), CKWC_PLUGIN_VERSION );
 				break;
 
-			/**
-			 * Settings Screen.
-			 */
-			case 'settings':
-			default:
-				// Enqueue Select2 CSS.
-				ckwc_select2_enqueue_styles();
```

Comment:
This was never needed, as we didn't use select2 on the settings screen prior to this PR. We now use it, but WooCommerce loads its own version of the select2 library for us.